modules/systemd-journal/journald-subsystem.c

```diff
@@ -57,20 +57,25 @@ int (*sd_id128_get_boot)(sd_id128_t *ret);
 
 #define JOURNAL_LIBRARY_NAME "libsystemd-journal.so.0"
 #define SYSTEMD_LIBRARY_NAME "libsystemd.so.0"
+#define ID128_LIBRARY_NAME   "libsystemd-id128.so"
 
 const char *journald_libraries[] = {JOURNAL_LIBRARY_NAME, SYSTEMD_LIBRARY_NAME, NULL};
+const char *journald_ext_libraries[] = {ID128_LIBRARY_NAME, JOURNAL_LIBRARY_NAME, SYSTEMD_LIBRARY_NAME, NULL};
 
 static GModule *journald_module;
+static GModule *journald_ext_module;
```

Comment:
we could probably move these to local variables in load_journald_subsystem() and then pass them along
using arguments to _load_journald_subsystem()

Reply:
yeah, it came to my mind as well, the only reason I left it this way is that once we might want to add a really proper cleanup then we might want to call unload_journald_subsystem() and that would not be that nice to pass these to that function too